src/pages/ChemicalDetails.vue

```diff
@@ -35,15 +41,27 @@ export default {
 
   },
   computed: {
+    
+    chemData: function() {
```

Comment:
Not sure if this is how it's sposed to work but - wouldn't this function be a perfect example of a store action? I feel like we'll end up reusing this in a lot of places.

Reply:
i can look into converting this into a store action

Reply:
done